components/Slideshow.vue

```diff
@@ -151,23 +158,60 @@
     return parsedPayload.success ? parsedPayload.data : defaultPayload
   })
 
+  /**
+   * Start: set swiper options
+   */
+
+  const progressValue = ref(0)
+
+  const isLoop = computed(() => {
+    return validatedPayload?.value?.images?.length > 3 ? true : false
```

Comment:
minor: I think we don't have to explicitly return boolean here @doohanas 

```suggestion
    return validatedPayload?.value?.images?.length > 3
```